examples/showcase/xkcd.py

```diff
@@ -9,6 +9,8 @@
 import numpy as np
 
 with plt.xkcd():
+    
+###############################################################################    
```

Comment:
I wonder if the unindented comment inside the `with` block is causing the error on CircleCI.

Reply:
Not exactly; the `###` splits up the file, so the `with` above is empty. This `###` needs to go above the `with` context.